test/components/connect.spec.js

```diff
@@ -329,6 +329,135 @@ describe('React', () => {
       done()
     })
 
+    it('should passthrough value of function identity requests after invoking function', (done) => {
+      @connect(() => ({ testFetch: { value: () => Promise.resolve('foo'), meta: { test: 'voodoo' } } }))
+      class Container extends Component {
+        render() {
+          return <Passthrough {...this.props} />
+        }
+      }
+
+      const container = TestUtils.renderIntoDocument(
+        <Container />
+      )
+
+      const stub = TestUtils.findRenderedComponentWithType(container, Passthrough)
+      expect(stub.props.testFetch).toIncludeKeyValues({
+        fulfilled: false, pending: true, refreshing: false, reason: null, rejected: false, settled: false, value: null, meta: { test: 'voodoo' }
+      })
+
+      setImmediate(() => {
+        const stub = TestUtils.findRenderedComponentWithType(container, Passthrough)
+        expect(stub.props.testFetch).toIncludeKeyValues({
+          fulfilled: true, pending: false, refreshing: false, reason: null, rejected: false, settled: true, value: 'foo', meta: { test: 'voodoo' }
+        })
+
+        done()
+      })
+    })
+
+    it('should invoke value() only if `comparison` changed', (done) => {
```

Comment:
test name dupe. this looks like this was an accidential copy/paste of another test